contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1234,19 +1245,21 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     )
         internal
     {
+        // Start by checking for a load as we only want to charge nuisance gas proportional to
+        // contract size once.
+        _checkAccountLoad(_address);
+
         // Check whether the account has been changed before and mark it as changed if not. We need
         // this because "nuisance gas" only applies to the first time that an account is changed.
         (
             bool _wasAccountAlreadyChanged
         ) = ovmStateManager.testAndSetAccountChanged(_address);
 
-        // If we hadn't already changed the account, then we'll need to charge "nuisance gas" based
-        // on the size of the contract code.
+        // If we hadn't already changed the account, then we'll need to charge a constant
+        // "nuisance gas" that covers the cost of updating the account state.
         if (_wasAccountAlreadyChanged == false) {
             ovmStateManager.incrementTotalUncommittedAccounts();
-            _useNuisanceGas(
-                (Lib_EthUtils.getCodeSize(_getAccountEthAddress(_address)) * NUISANCE_GAS_PER_CONTRACT_BYTE) + MIN_NUISANCE_GAS_PER_CONTRACT
-            );
+            _useNuisanceGas(MIN_NUISANCE_GAS_PER_CONTRACT);
```

Comment:
Same thing here as with nonce fix -- this is audit L01, can we break into separate PR for convenience?  Happy to do it myself if you need it real quick.

Reply:
Ah ok, will revert the change.